### DISCUSSION TITLE
Can VK_KHR_xlib_surface requirement be disabled?

### DISCUSSION THREAD

**Discussion by McCallisterRomer** [DiligentGraphics/DiligentEngine#319] [Q&A]
It looks like both VK_KHR_xlib_surface and VK_KHR_xcb_surface are required extensions on Linux. Is there any way I can target
only xcb/make xlib optional? Use case here is non-critical, but I'm trying to reuse a test setup made for a different vulkan
engine. It uses swiftshader, which only supports the VK_KHR_xcb_surface extension.

**Comment by TheMostDiligent** [comment]
Yes, enabling both extensions is not required.
The instance can check if the extension is supported - this should not be too difficult to add.
